6_visualize.yml

```diff
@@ -43,6 +44,9 @@ targets:
       tmp=I('6_visualize/tmp'),
       log=I('6_visualize/log'))
 
+  image_placement:
+    command: viz_config[[I('image_placement')]]
```

Comment:
It'd be a good idea to subset this even further and only pass to `datetime_fun()` those parameters strictly needed for datetime. Otherwise every time someone nudges the legend around (for example), they'll also end up rebuilding the datetime step, which would be inefficient.

Reply:
OK will do

Reply:
remake doesn't like `command: viz_config[[I('image_placement')]][[I('datetime')]]`, any reason why I can't do that? I get an error
 ```
Error in FUN(X[[i]], ...) : While processing target 'datetime_placement':
    Unknown special function [[
```
should I just create `datetime_placement` in `viz_config.yml` rather than indexing here?

Reply:
I created a new `datetime_placement` in `viz_config.yml` for now and index to that

Reply:
was in meeting. new datetime_placement in viz_config is fine. there might be a purrr command for this sort of thing, but the main issue with the command remake didn't like is that remake only allows one function per command. two calls to `[[` is one call too many for remake. so if you wanted to use a nested config structure and then pull out just the datetime bit, you'd need to have two targets - one would pull out all the image placement, and the next would pull out the datetime-specific params from the image placement object.

but to reiterate, the route you chose is just fine.